guide/getting-started/common-user-flows.md

```diff
@@ -44,8 +44,14 @@ Some applications are like Swiss Army Knives. They provide many different indivi
 
 Let's take a look at some common user needs and workflows that can be created for them.
 
-1. Table of contents
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
```

Comment:
what does this line do since it doesn't get outputted on the page? is it for screen readers?

Reply:
It tells the table of contents script whether to display an ordered or unordered list. `1. Table of contents` would create an ordered one.